app/containers/Encrypt/Encrypt.jsx

```diff
@@ -1,61 +1,50 @@
 // @flow
 import React from 'react'
-import { noop } from 'lodash-es'
-import { wallet } from '@cityofzion/neon-js'
-import { wallet as n3Wallet } from '@cityofzion/neon-js-next'
+import { useIntl } from 'react-intl'
 
-import EncryptPanel from '../../components/Settings/EncryptPanel'
+import { ROUTES } from '../../core/constants'
+import FullHeightPanel from '../../components/Panel/FullHeightPanel'
+import EncryptForm from '../../components/Settings/EncryptForm'
+import EncryptSuccess from '../../components/Settings/EncryptSuccess'
 import LockIcon from '../../assets/icons/lock.svg'
+import CloseButton from '../../components/CloseButton'
 import styles from './Encrypt.scss'
 
 type Props = {
-  generateNewEncryptedWIF: Function,
-  validatePassphraseLength: Function,
-  chain: string,
+  resetEncryptedWIF(): void,
+  encryptedWIF: string,
 }
 
-type State = {
-  n3Key: string,
-}
-
-export default class Encrypt extends React.Component<Props, State> {
-  static defaultProps = {
-    validatePassphraseLength: noop,
-    isWIF: noop,
-  }
-
-  state = {
-    n3Key: '',
-  }
-
-  generateNewN3EncryptedWif = async (wif: string, passphrase: string) => {
-    const n3Key = await n3Wallet.encrypt(wif, passphrase)
-    this.setState({ n3Key })
-  }
-
-  render = () => {
-    const {
-      generateNewEncryptedWIF,
-      validatePassphraseLength,
-      chain,
-    } = this.props
-    return (
-      <div className={styles.encrypt}>
-        <EncryptPanel
-          title="Encrypt Private Key"
-          handleSubmit={generateNewEncryptedWIF}
-          handleN3Submit={this.generateNewN3EncryptedWif}
-          isWIF={chain === 'neo3' ? n3Wallet.isWIF : wallet.isWIF}
-          validatePassphraseLength={validatePassphraseLength}
-          n3Key={this.state.n3Key}
-        />
-      </div>
-    )
-  }
-
-  renderIcon = () => (
-    <div>
-      <LockIcon />
-    </div>
+const HeaderIcon = () => (
+  <div>
+    <LockIcon />
+  </div>
+)
+
+const Encrypt = ({ encryptedWIF, resetEncryptedWIF }: Props) => {
+  const intl = useIntl()
+
+  return (
+    <FullHeightPanel
+      className={styles.encrypt}
+      headerText={intl.formatMessage({ id: 'encryptPanelHeader' })}
+      renderCloseButton={() => (
+        <CloseButton onClick={resetEncryptedWIF} routeTo={ROUTES.SETTINGS} />
+      )}
+      renderHeaderIcon={HeaderIcon}
+      renderInstructions={
+        encryptedWIF
+          ? () => intl.formatMessage({ id: 'encryptInstructions' })
```

Comment:
We should skip the instructions part OR make the instructions say something different than "Choose a passphrase to encrypt an existing key" like "please save your information in a safe place" once they have successfully encrypted a key (at this point they have already chosen a passphrase)

<img width="645" alt="Screen Shot 2022-09-01 at 10 28 35 AM" src="https://user-images.githubusercontent.com/13072035/187965738-ba05f342-95cf-40c4-b021-2085ae0356c8.png">

Reply:
Ok. You are right. I will fix.